sdk/objectanchors/Azure.MixedReality.ObjectAnchors/tests/ObjectAnchorsClientLiveTest.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Numerics;
+using System.Reflection;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.MixedReality.Authentication;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using NUnit.Framework;
+
+namespace Azure.MixedReality.ObjectAnchors.Tests
+{
+    public class ObjectAnchorsClientLiveTest : RecordedTestBase<ObjectAnchorsClientTestEnvironment>
+    {
+        private const string assetsFolderName = "Assets";
+        private const string assetsFileName = "switchgear02_obj.obj";
+        private const string modelDownloadFileName = "switchgear02_obj_model.ply";
```

Comment:
I don't see these files being added.

Reply:
Gah... obj is a file format in gitignore, need to consciously add the file.

Reply:
Added

Reply:
The ply file is the file to be created